### PR TITLE
Adds thanos downsample

### PR DESCRIPTION
**Pull request by jalev**
Since we need to start using downsample, this PR sets it up for us. Fortunately a lot of the configuration has already been set up, and this is mostly a C&P job.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -12,6 +12,9 @@ thanos_compactor_http_port: 19192
 thanos_compactor_data_dir: ""
 thanos_compactor_config_file: "/etc/thanos-compactor.yaml"
 
+thanos_downsample_config_file: "/etc/thanos-downsample.yaml"
+thanos_downsample_data_dir: ""
```
Comment (tw-peeterkarolin) [inline defaults/main.yml]: I don't see this being used anywhere.

File: templates/thanos-downsample-defaults.j2 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+THANOS_DOWNSAMPLE_CONFIG_FILE="--objstore.config-file {{ thanos_downsample_config_file }}"
+THANOS_DOWNSAMPLE_LOG_LEVEL="--log.level {{ thanos_downsample_log_level }}"
```
Comment (tw-peeterkarolin) [inline templates/thanos-downsample-defaults.j2]: This should also be defined in defaults and readme.

File: defaults/main.yml (hunk 1 above)
Comment (jalev) [reply defaults/main.yml]: Fixed.

File: templates/thanos-downsample-defaults.j2 (hunk 2 above)
Comment (jalev) [reply templates/thanos-downsample-defaults.j2]: Added. Was already in the README, just forgot about it.

Comment (tw-peeterkarolin) [reply templates/thanos-downsample-defaults.j2]: There is still no downsample specific log_level in defaults and readme, only for other components?

**Pull request merged**